### DISCUSSION TITLE
How to achieve dependencies between tasks

### DISCUSSION THREAD

**Discussion by Lenchok** [apache/airflow#23032] [Q&A]
### Apache Airflow version

2.2.2

### What happened

My requirement is that the post-task will run only if the pre-dependency runs successfully. If the pre-dependency does not run or fails, the post-task will not run even when the time is up. How to implement this function?

### Operating System

null

### Deployment

Other

### Deployment details

null

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
When you opened and issu, you have been adviced not to open issue when you have questions. Please next time read what the advice tells  you.

**Comment by potiuk** [comment]
Look at our docs and search for "task dependencies". You will find answer on your own without unnecessrily engaging other people. The docuementation we have is pretty comprehensive: https://airflow.apache.org/docs/
